index.Rmd

```diff
@@ -37,7 +37,7 @@ library(plyr)
 knitr::opts_chunk$set(cache=TRUE)  # cache the results for quick compiling
 ```
 
-## Download and clean all required data
+## Download and clean all required data 
 ```{r, message=FALSE, warning=FALSE, results = 'hide'}
 NY <- get_acs(geography = "county", 
```

Comment:
this is where you should also get the population estimates.  And you may want to limit this to some ages (18-30)?  Do you know how old the people who died were?  Maybe use a similar range?